test/new-e2e/cws/e2e_cws_test.go

```diff
@@ -0,0 +1,150 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Can you move `cws` to `test/new-e2e/tests/cws` ? We will move other teams folders later on